taccsite_cms/templates/guides/getting_started.html

```diff
@@ -0,0 +1,440 @@
+            with a verification link. Please click on the link to verify your
+            account.
+          </p>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="row">
+    <div class="col">
+      <h3>Setting up Multi-factor Authentication</h3>
+
+      <p>
+        Your account will need to have Multi-factor Authentication (MFA) setup
+        using the TACC Token App, available on the Google Play and iTunes
+        stores.
+      </p>
+
+      <div class="row">
+        <div class="col">
+          <p>
+            <img
+              class="img-fluid"
+              src="https://utrc.tacc.utexas.edu/media/filer_public_thumbnails/filer_public/6d/bd/6dbdaa04-dc27-4922-a446-f11bffa08189/mfa_required.png__525x144_q85_subsampling-2.png"
+            />
+          </p>
+        </div>
+
+        <div class="col">
+          <p>
+            If you do not have MFA pairing through the TACC Token App, click the
+            <strong>Click here for more info</strong> link.
+          </p>
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="col">
+          <p>
+            <img
+              class="img-fluid"
+              src="https://utrc.tacc.utexas.edu/media/filer_public_thumbnails/filer_public/c4/89/c48958f4-078d-4417-9a27-9361cef49129/mfa_pairing_dialog.png__525x290_q85_subsampling-2.png"
+            />
+          </p>
+        </div>
+
+        <div class="col">
+          <p>
+            In the dialog that appears, click the link to go to
+            <a href="https://portal.tacc.utexas.edu/account-profile/-/profile/view" target="_blank">https://portal.tacc.utexas.edu/account-profile/-/profile/view</a
+            >
+            .
+          </p>
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="col">
+          <p>
+            <img
+              class="img-fluid"
+              src="https://utrc.tacc.utexas.edu/media/filer_public_thumbnails/filer_public/7a/d8/7ad8efab-0527-434a-9e7c-45d19c96b627/fig-41-otp.png__525x350_q85_subsampling-2.png"
+            />
+          </p>
+        </div>
+
+        <div class="col">
+          <p>
+            You will be redirected to the TACC User Portal, where you can pair
+            your account with the TACC Token App.
+          </p>
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="col">
+          <p>
+            <img
+              class="img-fluid"
+              src="https://utrc.tacc.utexas.edu/media/filer_public_thumbnails/filer_public/b6/bb/b6bbc465-1666-4e45-915b-4152dac88701/confirm_pairing.png__525x132_q85_subsampling-2.png"
+            />
+          </p>
+        </div>
+
+        <div class="col">
+          <p>
+            Once you have completed MFA pairing with the TACC Token app, return to the Portal
+            <a href="https://utrc.tacc.utexas.edu/onboarding/setup" target="_blank">https://utrc.tacc.utexas.edu/onboarding/setup</a>
```

Comment:
Same here, any way to make this relative to the current portal?

Reply:
Yes. Done. See my [reply to original request](https://github.com/TACC/Core-CMS/pull/152#discussion_r598023843). Also fixed the URL to be accurate.